spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -114,6 +118,14 @@ func TestSample(t *testing.T) {
 	defer cleanup()
 
 	var out string
+
+	instanceID := fmt.Sprintf("go-sample-test-%s", uuid.New().String()[:8])
```

Comment:
Could we create another test for this? Or do they all need to run sequentially in the same test? It might be nice to move away from a single giant test.

Reply:
Thanks for suggestion. I've made it a separate test. I would like to make the task of refactoring the giant test into a separate PR.

Reply:
Sounds good! I don't think it's urgent -- just nice to have.

Looks like the test is failing right now:
```
integration_test.go:372: failed to create an instance: could not create instance projects/golang-samples-tests-7/instances/go-sample-test-afc7957b: rpc error: code = InvalidArgument desc = Invalid CreateInstance request.
```